tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
```

Comment:
config is more or less intuitive, maybe some of parameters worth commenting? or it's a usual ts config?

Reply:
This is default ts config with a few changes:
1. Strict check for implicit any has been disabled (to avoid marking all untyped code with the word "any")
2. Added path alias to allow write absolute imports like "import Button from 'components/Button"